offline/cable_LUC_EXPT.F90

```diff
@@ -745,11 +784,6 @@ subroutine READ_ClimateFile(LUC_EXPT)
     LUC_EXPT%ivegp = climate%iveg
     LUC_EXPT%biome = climate%biome
 
-    ! non-woody potential vegetation not considered to undergo LU change
```

Comment:
Good to see you're happy with removing this condition!

Reply:
That was very misleading actually. Good riddance!